examples/blueprint-construct/index.ts

```diff
@@ -173,6 +173,33 @@ export default class BlueprintConstruct {
             // Commenting due to conflicts with `CloudWatchLogsAddon`
             // new blueprints.AwsForFluentBitAddOn(),
             new blueprints.FluxCDAddOn(),
+            new blueprints.GpuOperatorAddon({
+                values:{
+                    driver: {
+                      enabled: true
+                    },
+                    mig: {
+                      strategy: 'mixed'
+                    },
+                    devicePlugin: {
+                      enabled: true,
+                      version: 'v0.13.0'
+                    },
+                    migManager: {
+                      enabled: true,
+                      WITH_REBOOT: true
+                    },
+                    toolkit: {
+                      version: 'v1.13.1-centos7'
```

Comment:
is this a mandatory parameter? I am curious how customers (and us) will update this moving forward.

Reply:
This toolkit version was verified as working with AL2 by Re Alvarez